src/page-content/navigation/header.json

```
@@ -14,6 +14,11 @@
           "to": "/cli"
         },
         {
+          "title": "Multi-factor Authentication",
+          "description": "MFA",
```

```suggestion
          "description": "Based on proven standards",
```

This is just a suggestion, but can we find something more expressive than "MFA" ?

---

>Secure login and authentication

this I like the best, committing it now, but feel free to change if you find something better.

---

Cool. Looks good there!